customers/schema.py

```diff
@@ -64,6 +64,20 @@ class Meta:
     invoicing_type = InvoicingTypeEnum()
     comment = graphene.String()
 
+    @classmethod
+    @login_required
+    def get_node(cls, info, id):
+        node = super().get_node(info, id)
+        if not node:
+            return None
+
+        user = info.context.user
+        # TODO: implement proper permissions
+        if node.user == user or user.is_superuser:
+            return node
+        else:
+            raise GraphQLError(_("You do not have permission to perform this action."))
```

Comment:
Should we use [VenepaikkaGraphQLError](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/berth_reservations/exceptions.py#L4) instead of plain GQLError?

Reply:
we should, good catch! 👍